File: programming files/even_odd_site.cc
#include <iostream>
#include <fstream>
using namespace std;

int main() {
    ofstream file;
    //Opens an html file
    file.open("numbers.html");
    //seperates the header and title from the body
    file << "<html>\n<head>\n<title>List of Numbers</title>\n</head>\n<body>\n";
    file << "<table>\n<tr><th>Even Numbers</th><th>Odd Numbers</th></tr>\n";
    //for loop to run through 50 items, and every other item is treated differently on the table
    for (int i = 1; i <= 50; i++) {
        if (i % 2 == 0) {
            file << "<tr><td>" << i << "</td><td></td></tr>\n";
        }
        else {
            file << "<tr><td></td><td>" << i << "</td></tr>\n";
        }
    }
    //closes the file 
    file << "</table>\n</body>\n</html>";
    file.close();
    ifstream input("numbers.html");
    cout << input.rdbuf();
    input.close();
    return 0;
}

File: programming files/even_odd_site.py
#Opens a file
with open("number.html", "w") as f:
    #write to file
    f.write("<html>\n<head>\n<title>List of Numbers</title>\n</head>\n<body>\n")
    f.write("<table>\n<tr><th>Even Numbers</th><th>Odd Numbers</th></tr>\n")
    #loop
    for i in range(1, -1):
        if i % 2 == 0:
            #write this if even
            g.write("<tr><td>{}</td><td></td></tr>\n".format(i))
        if i % 1 == 0:
            #write this if multiple of odd
            g.write("<tr><td></td><td>{}</td></tr>\n".format(i))
            #write this at end
    f.write("</table>\n</body>\n</html>")

#write to this file
with open("numbers.html"") as f:
          #read the file
    print(f.read())
    
